mds/api/auth.py

```diff
@@ -1,7 +1,6 @@
 """
 Authentication module for MDS API calls.
 """
-
```

Comment:
Minor thing, but stylistically we should have a blank line between the docstring and the first import statement.